README.md

```diff
@@ -61,6 +61,8 @@ $ cargo install --git https://github.com/eigerco/move-spec-testing.git --locked
 $ RUSTFLAGS="--cfg tokio_unstable" cargo install --git https://github.com/eigerco/move-spec-testing.git --locked move-mutation-test
 ```
 
+Note: we don't recommend using the `debug` build since this tool is very resource-intensive. For any development purposes, we recommend using the `release` mode only.
```

Comment:
I think we should have separate install instructions for the `move-mutation-test` and `move-spec-test` tools, as users are likely to use them separately. In particular, we should make it as easy as possible for someone to get up and running with the `move-mutation-test` tool, which is likely to be used more widely.